### DISCUSSION TITLE
Is there a way to monitor a group?

### DISCUSSION THREAD

**Discussion by SkyMoCo** [simplyequipped/pyjs8call#2] [Q&A] [answered]
I see you can monitor spots, but how about groups?

**Comment by SkyMoCo** [comment, marked as answer]
Right there in the example....
# watch a group
js8call.spots.add_group_watch('@AMRRON')

**Reply by simplyequipped** [reply]
Glad you found what you were looking for!

Here are some links to related documentation in case they are useful:

[Add group watch](https://simplyequipped.github.io/pyjs8call/pyjs8call/spotmonitor.html#SpotMonitor.add_group_watch)
[Remove group watch](https://simplyequipped.github.io/pyjs8call/pyjs8call/spotmonitor.html#SpotMonitor.remove_group_watch)
[Get list of watched groups](https://simplyequipped.github.io/pyjs8call/pyjs8call/spotmonitor.html#SpotMonitor.get_watched_groups)
[Set group spot callback](https://simplyequipped.github.io/pyjs8call/pyjs8call/client.html#Callbacks.group_spot)
